css/style.css

```diff
@@ -69,3 +68,45 @@ body {
   justify-content: space-between;
   margin: 0 206px;
 }
+/* footer */
+.footer {
+  background: #10367c;
+  height: 630px;
+}
+
+.footer-about {
+  display: flex;
+}
+
+.about {
+  margin: 120px 180px 120px 120px;
+}
+
+.property {
+  margin: 120px 284px 0 0;
+}
+
+li {
+  list-style: none;
+  margin-bottom: 24px;
+  font-size: 20px;
+  line-height: 23px;
+  color: #ffffff;
+  font-family:'Roboto-Light';
+}
+
+.otdelni {
+  padding: 0;
+}
+
+.triphouse {
+  margin: 0 120px;
+  font-size: 16px;
+  line-height: 16px;
+  color: #ffffff;
+  font-family:'Roboto-Light';
+}
+
+.bold {
+  font-weight: bold;
```

Comment:
This doesn't work clearly. You should use `font-weight: 500;` like you've write in `@font-face`